src/frontend/src/flows/addDevice/manage/addDevicePickAlias.ts

```diff
@@ -1,5 +1,9 @@
 import { html, render } from "lit-html";
 import { initLogout, logoutSection } from "../../../components/logout";
+// Regex Pattern for input: All characters, including starting(^) single character or more(+),
+// must be alphabet, number or underscore. Can have hyphen(s) or space(s) in the middle.
```

Comment:
any particular reason we allow starting with an underscore but not a hyphen? how about treating underscores like we do whitespaces and hyphens?